### PR TITLE
Prepare covid_act_now for staging

### PR DESCRIPTION
**Pull request by korlaxxalrok**
### Description
Feature:

Add params j2 template with S3 variables.

### Changelog
- Add ansible/templates/covid_act_now-params-prod.json.j2

### Fixes 
- Fixes #(issue)

### CODE REVIEW

**Comment by korlaxxalrok** [comment]
@eujing I think this covers the barest of necessities.

File: ansible/templates/covid_act_now-params-prod.json.j2 [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "static_file_dir": "./static",
+  "export_dir": "/common/covidcast/receiving/covid_act_now",
+  "cache_dir": "./cache",
+  "wip_signal": true,
```
Comment (eujing) [inline ansible/templates/covid_act_now-params-prod.json.j2:5]: Oops this is my fault for providing an outdated params.json, but the `wip_signal` key is no longer needed.
Sorry about that!

**Review by eujing — CHANGES_REQUESTED** [review]
Looks good to me, just the extra `wip_signal` I accidentally sent!

File: ansible/templates/covid_act_now-params-prod.json.j2 (hunk 1 above)
Comment (korlaxxalrok) [reply ansible/templates/covid_act_now-params-prod.json.j2]: Makes sense. I'll remove it.

**Review by eujing — APPROVED** [review]

**Pull request merged**